packages/template-retail-react-app/app/components/promo-popover/index.jsx

```diff
@@ -56,25 +56,25 @@ const PromoPopover = ({header, children, ...props}) => {
                         })}
                     />
                 </PopoverTrigger>
-                <Portal>
-                    <PopoverContent border="none" borderRadius="base">
-                        <Box boxShadow="lg" zIndex="-1">
-                            <PopoverArrow />
-                            <PopoverCloseButton />
-                            <PopoverHeader borderBottom="none">
-                                {header || (
-                                    <Text fontWeight="bold" fontSize="md">
-                                        <FormattedMessage
-                                            defaultMessage="Promotions Applied"
-                                            id="promo_popover.heading.promo_applied"
-                                        />
-                                    </Text>
-                                )}
-                            </PopoverHeader>
-                            <PopoverBody pt={0}>{children}</PopoverBody>
-                        </Box>
-                    </PopoverContent>
-                </Portal>
+                <PopoverContent border="none" borderRadius="base">
+                    <Box boxShadow="lg">
+                        <PopoverArrow />
+                        <PopoverCloseButton />
+                        <PopoverHeader borderBottom="none" tabindex="0">
```

Comment:
```suggestion
                        <PopoverHeader borderBottom="none">
```
The popover header is not an interactive element, so it should not have a tab index. VoiceOver reads the tooltip content, even without the tabindex:

![Screenshot of VoiceOver reading "Close, button, Promotions Applied description, Free Shipping Amount Above 50, tooltip"](https://github.com/SalesforceCommerceCloud/pwa-kit/assets/62956339/e3e32e24-6593-4f6a-bcc3-00a5b5dd8b7c)